e2e-tests/historical/endpoints/asset-hub-westend/accounts/asset-balances/index.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright 2017-2022 Parity Technologies (UK) Ltd.
+// This file is part of Substrate API Sidecar.
+//
+// Substrate API Sidecar is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+// import accountAssetBalance450000 from './450000.json';
+// import accountAssetBalance650000 from './650000.json';
+// import accountAssetBalance960000 from './960000.json';
+// import accountAssetBalance986791 from './986791.json';
+// import accountAssetBalance1100000 from './1100000.json';
+// import accountAssetBalance1300000 from './1300000.json';
+import accountAssetBalance1800000 from './1800000.json';
+
+export const assetHubWestendAccountAssetBalanceEndpoints = [
+	// [
```

Comment:
There is a bunch of commented out e2e-tests here. Up to you if you think they hold merit, but just having v700 is more than enough for me atleast.

Reply:
Sorry, I thought I got all of them, it's done now.